server/routes/redirects.py

```diff
@@ -101,4 +106,17 @@ def insights():
   return redirect(
       url_for('explore.page',
               _scheme=current_app.config.get('SCHEME', 'https'),
-              code=302))
\ No newline at end of file
+              code=302))
+
+
+def load_redirects(name):
+  local_file = gcs.download_file(bucket=GLOBAL_CONFIG_BUCKET,
+                                 filename='redirects.json')
+  with open(local_file) as fp:
+    mapping = json.load(fp)
+    return mapping.get(name, '/')
+
+
+@bp.route('/link/<path:name>')
+def link(name):
+  return redirect(load_redirects(name), code=302)
```

Comment:
Will this download and load the file from GCS for every link call?

Reply:
Yes, it's doing that to make sure we can dynamically change the content. The performance seems ok.